src/main/java/com/pst/login/service/LoginService.java

```diff
@@ -26,6 +26,8 @@ public class LoginService {
     private String messageServiceUrl;
 
     private final String updateUserUrl = "/update-user/";
+    
+    private String password;
```

Comment:
Check once are you using this variable or not and remove it

Reply:
Using man

Reply:
okay fine